.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: objective-c
+osx_image: xcode8.1
+rvm:
+  - 2.2.3
+
+env:
+  CODE_SIGNING_REQUIRED: NO
+
+notifications:
+  email: false
+
+xcode_project: Retired.xcodeproj
+xcode_scheme: Retired iOS
+xcode_sdk: iphonesimulator10.1
+
+before_install:
+  - brew update
+  - brew outdated carthage || brew upgrade carthage
```

Comment:
yeah -- it's in the docs for travis: https://docs.travis-ci.com/user/osx-ci-environment/#Homebrew (this was causing failure previously)